config/boards/orangepizero2w.csc

```diff
@@ -0,0 +1,15 @@
+# Allwinner H618 quad core 1GB/1.5GB/2GB/4GB RAM
+BOARD_NAME="Orange Pi Zero2W"
+BOARDFAMILY="sun50iw9"
+BOARD_MAINTAINER="chraac"
+BOOTCONFIG="orangepi_zero2w_defconfig"
+OVERLAY_PREFIX="sun50i-h616"
+BOOT_LOGO="desktop"
+BOOTBRANCH="tag:v2024.04-rc5"
+BOOTDIR="u-boot-zero2w"
+BOOTPATCHDIR="v2024.04-rc5-zero2w"
+KERNEL_TARGET="current,edge"
+FORCE_BOOTSCRIPT_UPDATE="yes"
+KERNEL_DRIVERS_SKIP+=(driver_rtw88)
```

Comment:
Why do we need to skip rtw88 driver here? IIRC this board uses uwe5622 wireless not realtek ones

Reply:
The only concern I had was that some boards has rtw88 driver and the 64-bit allwinner current and edge kernel are same across all boards. So adding this will create 2 different 64-bit kernels with same package names which can get confusing for the apt repo.

Its possible that the issue you experienced is already fixed and you just need to rebase the branch. So give that a try.

Reply:
okay, gonna remove it and try to build again